app/src/main/java/com/example/parkingsystemjava/mvp/view/ParkingReservationView.java

```diff
@@ -2,22 +2,47 @@
 
 import android.app.Activity;
 import android.app.FragmentManager;
+import android.content.Context;
+import android.widget.Toast;
+import com.example.parkingsystemjava.R;
+import com.example.parkingsystemjava.databinding.ActivityReservationBinding;
 import com.example.parkingsystemjava.fragments.DateReservationDialogFragment;
+import com.example.parkingsystemjava.listener.ListenerDateTime;
 import com.example.parkingsystemjava.mvp.contract.ParkingReservationContract;
 import com.example.parkingsystemjava.mvp.view.base.ActivityView;
 
 public class ParkingReservationView extends ActivityView implements ParkingReservationContract.ReservationActivityView {
 
     private static final String DIALOG_FRAGMENT_ENTRY_DATE_PICKER = "DIALOG_FRAGMENT_ENTRY_DATE_PICKER";
 
-    public ParkingReservationView(Activity activity) {
+    private final ActivityReservationBinding binding;
+
+    public ParkingReservationView(Activity activity, ActivityReservationBinding binding) {
         super(activity);
+        this.binding = binding;
     }
 
     @Override
-    public void showEntryDatePicker() {
-        DateReservationDialogFragment entryReservationDialogFragment = DateReservationDialogFragment.newInstance();
+    public void showDatePicker(ListenerDateTime listenerDateTime) {
+        DateReservationDialogFragment entryReservationDialogFragment = DateReservationDialogFragment.newInstance(listenerDateTime);
         FragmentManager fragmentManager = getFragmentManager();
         entryReservationDialogFragment.show(fragmentManager, DIALOG_FRAGMENT_ENTRY_DATE_PICKER);
     }
+
+    @Override
+    public void showDateSelected(String date) {
+        if (binding.editTextReservationActivityEntry.getText().toString().equals("")) {
```

Comment:
dont use harcodes, use constants.
or better, you can try with .isEmpty()

Reply:
done